Source/Core/Common/Config/Config.h

```diff
@@ -96,4 +96,15 @@ void SetBaseOrCurrent(const ConfigInfo<T>& info, const std::common_type_t<T>& va
   else
     Set<T>(LayerType::CurrentRun, info, value);
 }
-}
+
+// Used to suppress InvokeConfigChangedCallbacks until after the completion of many config changes.
+class ConfigChangeCallbackGuard
+{
+public:
+  ConfigChangeCallbackGuard();
+  ~ConfigChangeCallbackGuard();
+
+  ConfigChangeCallbackGuard(const ConfigChangeCallbackGuard&) = delete;
+  void operator=(const ConfigChangeCallbackGuard&) = delete;
```

Comment:
```suggestion
  ConfigChangeCallbackGuard& operator=(const ConfigChangeCallbackGuard&) = delete;
```

Reply:
I figured the `operator=` didn't need to be that sensible considering it's deleted. But I'll make the change.